src/elements/pages/access-token.ts

```diff
@@ -0,0 +1,40 @@
+import { LitElement, PropertyValues, TemplateResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { cssify } from '../../utils/css';
+import global from '../../utils/global';
+import routes from '../../routes';
+import logging from '../../utils/logging';
+import { globalEventGroups } from '../../utils/global-events';
+import RvtRouter from '../root/rvt-router';
+
+@customElement('page-access-token')
+export default class AccessTokenLink extends LitElement {
+	// static styles = cssify(styles);
```

Comment:
Can we remove this?

Reply:
https://github.com/rivet-gg/hub/pull/28/commits/4fe793737e7c070533ee26d38644c2d55a1796f3